components/locale/hr_HR.tsx

```diff
@@ -4,6 +4,8 @@ import TimePicker from '../time-picker/locale/hr_HR';
 import Calendar from '../calendar/locale/hr_HR';
 import { Locale } from '../locale-provider';
 
+const typeTemplate = '${label} nije valjan ${type}';
```

Comment:
Please disable `no-template-curly-in-string` lint error here.

Reply:
I did it, thanks!